src/Core/Application/Common/Specification/SpecificationBuilderExtensions.cs

```diff
@@ -79,29 +81,184 @@ public static ISpecificationBuilder<T> PaginateBy<T>(this ISpecificationBuilder<
         return new OrderedSpecificationBuilder<T>(specificationBuilder.Specification);
     }
 
-    private static void AddSearchPropertyByKeyword<T>(this ISpecificationBuilder<T> specificationBuilder, Expression propertyExpr, ParameterExpression paramExpr, string keyword)
+
+    private static MemberExpression GetMemberExpression(string propertyName, ParameterExpression parameter)

```

Comment:
looks like this method can also be used a bit higher up in the "SearchByKeyword" method

Reply:
fixed